compose/material3/material3/src/darwinMain/kotlin/androidx/compose/material3/PlatformDateFormat.darwin.kt

```diff
@@ -43,6 +43,7 @@ internal actual object PlatformDateFormat {
         val nsDate = NSDate.dateWithTimeIntervalSince1970(utcTimeMillis / 1000.0)
 
         return NSDateFormatter().apply {
+            setTimeZone(TimeZone.UTC.toNSTimeZone())
```

Comment:
This should have been part of https://github.com/JetBrains/compose-multiplatform-core/pull/877 but i forgot to commit it